api/src/test/java/marquez/OpenLineageIntegrationTest.java

```diff
@@ -390,6 +417,27 @@ public void testOpenLineageJobHierarchyAirflowIntegrationWithParentAndParentRunF
         .hasFieldOrPropertyWithValue("parentJobName", null);
     List<Run> runsList = client.listRuns(NAMESPACE_NAME, dagName);
     assertThat(runsList).isNotEmpty().hasSize(1);
+
+    marquez.common.models.JobId jobId =
+        new marquez.common.models.JobId(NamespaceName.of(NAMESPACE_NAME), JobName.of(dagName));
+    String nodeId = NodeId.of(jobId).getValue();
+    HttpRequest request =
+        HttpRequest.newBuilder()
+            .uri(URI.create(baseUrl + "/api/v1/lineage/direct?parentJobNodeId=" + nodeId))
+            .header("Content-Type", "application/json")
+            .GET()
+            .build();
+
+    HttpResponse<String> resp;
```

Comment:
The common practice for the tests in this class is to support newly implemented endpoints in Marquez java client and use java client to send api calls and deserialise responses, which allows more accurate tests.

Reply:
done